### DISCUSSION TITLE
[Unearth] Intermittently importing a model without a bentleyProjectId query param.

### DISCUSSION THREAD

**Discussion by gabe-unearth** [iTwin/itwinjs-core#2773] [Q&A]
I'm running into an issue where I'm importing a model and, when trying to open the viewer from our app, I get the error "missing query param: bentleyProjectID".  I remove the reference from my app, re-add and it works fine.  It's intermittent.  I'm also intermittently hitting failure to authenticate errors. Is this something you've observed before? Any leads as to why this is?

**Comment by aruniverse** [comment]
Could you provide us more information? 

What viewer? Are you using the itwin web viewer, bootstrapped from our cra template? If so, what's the version of the itwin/web-viewer-react package? What's the version of the itwinjs core packages?

Could you post a snippet of the code that references `bentleyProjectID`?
